src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const images = document.querySelectorAll('a');
+const mainImg = document.querySelector('#largeImg');
+
+const imagesArray = Array.from(images);
+
+imagesArray.forEach((image) => {
+  image.addEventListener('click', (el) => {
+    el.preventDefault();
+    mainImg.src = image.href;
```

Comment:
The code sets `mainImg.src` to `image.href`. Ensure that the `href` attribute of the anchor tags contains valid image URLs. If not, you may need to adjust the logic to extract the correct image URLs from the anchor tags.